README.md

```diff
@@ -1,3 +1,138 @@
 # PlayTools
 
-PlayTools from PlayCover
+PlayTools is an essential part of [PlayCover](https://github.com/PlayCover/PlayCover). PlayTools implements core functions of PlayCover, including display control, key mapping and bypassing.
+
+## Display Control
+
+<!-- iOS APPs running on macOS usually have fixed display settings, which may not be suitable for the user's needs. PlayTools allows you to adjust the display settings of the game, so that you can enjoy the game in a more comfortable way. -->
+
+PlayTools allows you to control:
+
+- Resolution: Supports 1080p, 4k, 1440p and custom resolution
+- Aspect Ratio: Supports 16:9, 16:10, 4:3 and custom aspect ratio
+- Scale Factor: Supports custom scale factor (e.g. 1.0, 1.5, 2.0)
+- Display Orientation: Supports manually rotating the game window during game play.
```

Comment:
May or may not be useful, but it also allows you to control:

- Application type (most useful case is changing to type `game` and the type reflects in screen time usage)
- Custom discord activity
- Device type

Reply:
They don't quite belong to "Display" though I'll just append these three to this list